Week2/solution week2/solution week2.ipynb

```diff
@@ -0,0 +1,481 @@
+    "    t=(sid,sname)\n",
+    "    l.append(t)\n",
+    "    t=(sid,scity,smono)\n",
+    "    a.append(t)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "487ab80f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q=\"insert into stud values(?,?)\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "764c8523",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<sqlite3.Cursor at 0x1b075d07bc0>"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cur.executemany(q,l)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "5d6caacb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q=\"insert into stud_details values(?,?,?)\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "d326511a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<sqlite3.Cursor at 0x1b075d07bc0>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cur.executemany(q,a)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "178ed811",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "con.commit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "2734a8b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q='''SELECT stud.name, stud_details.city\n",
+    "            FROM stud\n",
+    "            INNER JOIN stud_details ON stud.id = stud_details.id\n",
+    "            WHERE stud_details.city = 'surat' '''"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "7081f6c0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<sqlite3.Cursor at 0x1b075d07bc0>"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cur.execute(q)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "3a9c607b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r=cur.fetchall()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "4000b19b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "student name:om\n",
+      "student city:surat\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in r:\n",
+    "    print(\"student name:{}\".format(i[0]))\n",
+    "    print(\"student city:{}\".format(i[1]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "cb45c09a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q='''SELECT stud.name, stud_details.city\n",
+    "            FROM stud\n",
```

Comment:
Can we do the same query without using join...?

Reply:
q='''SELECT stud.name, stud_details.city
            FROM stud
            INNER JOIN stud_details ON stud.id = stud_details.id
            WHERE stud_details.city = 'surat' '''

I mean  this query......

Reply:
it is join query please check again.

Reply:
Yes but I said that without using join...